### DISCUSSION TITLE
GitHub Code Search unable to use operator

### DISCUSSION THREAD

**Discussion by GZLiew** [community/community#73094] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi I am trying to use the code search syntax operator 
as mentioned in your docs 
but I cant get it working and it always return empty result for me 
https://docs.github.com/en/search-github/github-code-search/understanding-github-code-search-syntax#using-boolean-operations

```
is:open AND is:pr AND NOT author:GZLIew
```

**Comment by ktauchathuranga** [comment]
try using this,
`is:open is:pr -author:GZLiew`
and that will search PR's that is NOT opened by you!
if you wanna see PR's that were opened by you try this,
`is:open is:pr author:GZLiew`
noticed the removed "-"

**Comment by look** [comment, marked as answer]
@GZLiew sorry for the confusion, but you are trying to search PRs with code search operators. The code search operators (and boolean logic like `AND` and `OR`) only support searching code.

**Reply by GZLiew** [reply]
Alright so the supported group key are `path:` and `language:` only ? Are there anything else?

**Reply by look** [reply]
Code search supports `repo:`, `org:`/`user:`, `path:`, `language:`, `symbol:`, `content:`, and `is:` (which supports various filters, like `archived`, `fork`), as [laid out in our documentation](https://docs.github.com/en/search-github/github-code-search/understanding-github-code-search-syntax#using-qualifiers).

Other search types have their own filters, for example, [Issues and PRs](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests).
